server.js

```diff
@@ -0,0 +1,28 @@
+const http = require('http');
+const url = require('url');
+const fs = require('fs');
+
+const server= http.createServer((req, res)=>{
+    const pathName = req.url;
+    if(pathName === '/'){
+        res.writeHead(200, {'Content-type': 'text/html'})
+        const indexPage = fs.readFileSync(`${__dirname}/pages/index.html`, 'utf-8')
+        res.end(indexPage)

```

Comment:
Route Handlers can be enclosed in functions that can be reused, this will reduce the lines of code 
Heres a sample Function 

## SAMPLE CODE 
// GET AND SEND HTML PAGE RESPONSE
function sendHtmlPageResponse(filePath,fileErrorMessage,res)
{

  fs.readFile(filePath,(err, data)=>{

    if(err)
    {
      console.log(fileErrorMessage)
      console.log(`err: ${ err }`)
      res.writeHead(500,{'Content-Type':'text/plain'})
      return res.end(' Error Occured while fetching page ')
    }

      res.writeHead(200,{'Content-Type':'text/html'})
      return res.end(data)

  })
}